documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -1,342 +1,462 @@
 ----
 
-====== For all 64-bit builds
+For 32-bit kernels, pass the image modules and DTB files like so, prefixing each image module with `z`:
 
-NOTE: Note the difference between Image target between 32- and 64-bit.
-
-[,bash]
+[source,console]
 ----
-make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- Image modules dtbs
+$ make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- zImage modules dtbs
 ----
 
-==== Install directly onto the SD card
+==== Install the kernel
 
-Having built the kernel, you need to copy it onto your Raspberry Pi and install the modules. This is best done directly using an SD card reader.
+Having built the kernel, you need to copy it onto your Raspberry Pi boot media (likely an SD card or SSD) and install the modules.
 
-First, use `lsblk` before and after plugging in your SD card to identify it. You should end up with something a lot like this:
+First, run `lsblk`. Then, connect your boot media. Run `lsblk` again; the new device represents your boot media. You should see output similar to the following:
 
 ----
 sdb
    sdb1
    sdb2
 ----
 
-with `sdb1` being the `FAT` filesystem (boot) partition, and `sdb2` being the `ext4` filesystem (root) partition.
+If `sdb` represents your boot media, `sdb1` represents the the `FAT32`-formatted **boot partition** and `sdb2` represents the (likely `ext4`-formatted) **root partition**.
 
-Mount these first, adjusting the partition letter as necessary:
+First, mount these partitions as `/mnt/boot` and `/mnt/root`, adjusting the partition letter to match the location of your boot media:
 
-[,bash]
+[source,console]
 ----
-mkdir mnt
-mkdir mnt/fat32
-mkdir mnt/ext4
-sudo mount /dev/sdb1 mnt/fat32
-sudo mount /dev/sdb2 mnt/ext4
+$ mkdir mnt
+$ mkdir mnt/boot
+$ mkdir mnt/root
+$ sudo mount /dev/sdb1 mnt/boot
+$ sudo mount /dev/sdb2 mnt/root
 ----
 
-NOTE: You should adjust the drive letter appropriately for your setup, e.g. if your SD card appears as `/dev/sdc` instead of `/dev/sdb`.
-
-Next, install the kernel modules onto the SD card:
-
-===== For 32-bit
+Next, install the kernel modules onto the boot media:
 
-[,bash]
+* For 64-bit kernels:
++
+[source,console]
 ----
-sudo env PATH=$PATH make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- INSTALL_MOD_PATH=mnt/ext4 modules_install
+$ sudo env PATH=$PATH make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- INSTALL_MOD_PATH=mnt/root modules_install
 ----
 
-===== For 64-bit
-
-[,bash]
+* For 32-bit kernels:
++
+[source,console]
 ----
-sudo env PATH=$PATH make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- INSTALL_MOD_PATH=mnt/ext4 modules_install
+$ sudo env PATH=$PATH make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- INSTALL_MOD_PATH=mnt/root modules_install
 ----
 
-Finally, copy the kernel and Device Tree blobs onto the SD card, making sure to back up your old kernel:
-
-===== For 32-bit
+Next, build and install the kernel, modules, and Device Tree blobs into the boot partition, backing up your original kernel.
```

Comment:
You've _already_ built the kernel and modules. We installed the modules in the previous `modules_install` step, so at this stage we're only installing the kernel and Device Tree blobs :slightly_smiling_face: